### ISSUE TITLE
Entity implements `device_state_attributes`

### ISSUE THREAD

**Issue by mdegat01** [edwork/homeassistant-peloton-sensor#22]
After updating Home Assistant to version `2021.12.1` with this integration installed I received the following warning in my log:

```
2021-12-13 11:50:37 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.peloton_mdegat01 (<class 'custom_components.peloton.sensor.PelotonSensor'>) implements device_state_attributes. Please report it to the custom component author.
```

So as requested I am reporting it. To my knowledge this warning just started appearing in version `2021.12.1`. I am not super diligent about checking my HA logs but I check them often enough that I feel like I would've noticed it if this was happening in version `2021.11`.

**Comment by RobertD502** [comment]
I opened a PR #21 that will fix this. Once @edwork pushes it through, the log message will disappear.

**Comment by edwork** [comment]
New Release Published - Version 0.6.0!.  

I've merged @RobertD502's fix and the logs are looking clean now!  

https://github.com/edwork/homeassistant-peloton-sensor/releases/tag/0.6.0
